addons/resource.language.en_gb/resources/strings.po

```diff
@@ -21275,3 +21278,9 @@ msgstr ""
 msgctxt "#39110"
 msgid "Select Resolution"
 msgstr ""
+
+#. audio flag - FLAG_ORIGINAL
```

Comment:
mind adding  something like `short/abbreviated version of "original language"` to the contextual help string to help our translators? Thanks.